package/Resources/ClientDefinitions.ini

```diff
@@ -15,7 +15,7 @@ GameLauncherExecutableName=syringe.exe
 ; Game executable name for Linux/Mac systems.
 UnixGameExecutableName=Resources/Compatibility/Unix/wine-game.sh
 ; Additional command line parameters applied after -SPAWN.
-ExtraCommandLineParams=-LOG -CD
+ExtraCommandLineParams=-LOG -CD -RA2ModeSaveID=0x8d143b95
```

Comment:
What is this for?

Reply:
https://github.com/CnCNet/yrpp-spawner/blob/40497cdd22d27fff38b902322f23cb52e4bdf209/src/Spawner/Ra2Mode.cpp#L117

This is a way to force the spawner to switch mode to RA2 when the game is loaded. At the moment, the client does not have a mechanism for processing saves from different game modes.
I could calculate this ID dynamically, on the spawner side, however Phobos (before version 0.3.0.2) has a bug due to which it also adds its id to the mod id instead of the engine id.

Reply:
I see, so that's a temporary workaround.